### DISCUSSION TITLE
Erik Bernhardsson - Modal

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#182] [Announcements]
- https://twitter.com/bernhardsson
- modal.com

## Intro

Andrew: Hello and welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey folks, our guest to today is Erik Bernhardsson. Erik is the CEO of Modal labs which we're excited to talk about today. Erik, before we start, would you like to tell our listeners a little more about yourself?

### Background

- [x] You built the music recommendation system for Spotify! That's so cool. What are some unique things you did to make the a good experience? Did it involve AI?

###  Modal

- [x] Why did you start modal? What type of work did you want to enable?
- [x] If I wanted to run this type of code before modal what would I have had to set up?
- [x] You seem to have put an incredible amount of effort into making the DX for modal great, why do you think it is so important to get right?
- [x] Do you consider Modal a server less platform?
- [x] What are some of the primitives the Modal provides that make cloud based distributed computing easier?
- [x] Can I spread the load of a cloud function over multiple containers?
- [x] You're approach to containerization is phenomenal and quite easy to use. Can you explain to our listeners how you configure it and why this is so important to the user feedback loop?
- [x] Other platforms are plagued by high costs and long build times. What steps have you all taken to make things fast? (cause it's really fast) How do you keep it so cheap for users?
- [x] What are some the of cool things that you've had to build to make this all work?
- [x] What types of cool things have you seen built on the platform?
- [ ] What you favorite feature of the platform? I personally love how easy it is to setup up web endpoints that "stay up" and have minimum running containers

### Future

- [x] What's next for modal? 
- [ ] How do you think AI is gonna change how we approach programming?

**Comment by just-be-dev** [comment]
Tooltips:
- https://www.quill-ui.com/
- https://github.com/ggerganov/ggwave

**Comment by hipstersmoothie** [comment]
https://microsoft.github.io/TypeChat/blog/introducing-typechat/
https://upstash.com/blog/qstash-announcement
